### DISCUSSION TITLE
error while resolving ssm parameter

### DISCUSSION THREAD

**Discussion by gtamas** [serverless/serverless#11234] [General]
So in my serverless.ts, I have manay of these kinda params:

```
 provider: {
    name: "aws",
    role: "${ssm:/Lambda/Something/roleArn~true}",
...
```

This worked fine with serverless 2 72 but does't seem to work with serverless 3 (latest version)

I'm getting this error when trying to run serverless-offline:

Cannot resolve variable at "provider.role": Parameter name: can't be prefixed with "ssm" (case-insensitive). If formed as a path, it can consist of sub-paths divided by slash symbol; each sub-path can be formed as a mix of letters, numbers and the following 3 symbols .-_

So removing the "~true" postfix solves it. But is this what I'm supposed to do? I thought this issue pops up only in case unsupported postfix (like ~false), but "true" and "split" used to be supported by v2. 

So is removing the psotfix OK or should I do something else?

Thx

**Comment by gtamas** [comment]
Hello?? Anybody? Should I open a bug issue instead? I don't think this is a bug, it seems to be more like a change in serverless 3. 
Any ideas?

**Comment by pgrzesik** [comment]
Hey @gtamas - the variable resolver changed in v3, you should be able to drop `~true` in your case and have it work correctly.

**Reply by gtamas** [reply]
OK thx!
